File: 백준/Bronze/2581. 소수/소수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

	public static void main(String[] args) throws IOException {

       BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
       
       int M = Integer.parseInt(br.readLine());
       int N = Integer.parseInt(br.readLine());
       int sum = 0;
       int minNum = N;
       
       
       for(int i = M; i <= N; i++) {
    	   int num = i;
    	   boolean isPrime = true;
    	   
    	   if(i == 1) {
    		   continue;
    	   }
		   for(int j = 2; j <= Math.sqrt(num); j++) {
			   if(i % j == 0) {
				   isPrime = false;
				   break;
			   }
			   
		   }
    		   
		   if(isPrime) {
			   sum += i;
			   
			   if(minNum > i) {
				   minNum = i;
			   }
		   }
		   
       }
       
       if(sum == 0) {
    	   System.out.println(-1);
       } else {
    	   System.out.println(sum);
    	   System.out.println(minNum);
       }

	
	}
}

File: 백준/Bronze/2581. 소수/소수.cc
#include <iostream>
#include <cmath>
using namespace std;

int main()
{	
	int m, n;
	
	cin >> m >> n;

	int sum = 0;
	int minNum = n;

	for (int i = m; i <= n; i++) {
		bool isPrime = 1;

		if (i == 1) continue;

		for (int j = 2; j <= sqrt(i); j++) {
			if (i % j == 0) {
				isPrime = 0;
				break;
			}
		}

		if (isPrime) {
			sum += i;

			if (minNum > i) minNum = i;
		}
	}

	if (sum == 0) {
		cout << -1;
	}
	else {
		cout << sum << "\n" << minNum;
	}
	
	return 0;
}